src/main/java/com/example/planit/utill/dto/DTOstudyPlanAndupComingStudySessionResponseToClient.java

```diff
@@ -0,0 +1,25 @@
+package com.example.planit.utill.dto;
+
+import com.example.planit.model.mongo.user.StudyPlan;
+import com.example.planit.model.studysession.StudySession;
+
+public class DTOstudyPlanAndupComingStudySessionResponseToClient extends DTOstatus {
```

Comment:
@oshriFadlon1  I think the name you've chosen to this class is not so readable. maybe think of a more short/ summerize name. 

Regardless, you did not camel-cased the "U" in ....Upcoming....

Reply:
chenge name to DTOstudyPlanAndSessionResponseToClient